lib/yard-sorbet/sig_handler.rb

```diff
@@ -80,8 +90,22 @@ def process
     docstring.add_tag(tag)
   end
 
+  sig do
+    params(sig_node: YARD::Parser::Ruby::MethodCallNode)
+      .returns(
+        {
+          abstract: T::Boolean,
+          params: T::Hash[String, T::Array[String]],
+          return: T.nilable(T::Array[String])
+        }
+      )
+  end
   private def parse_sig(sig_node)
-    parsed = {}
+    parsed = {
+      abstract: false,
+      params: {},
+      return: nil
+    }
     parsed[:abstract] = false
     parsed[:params] = {}
```

Comment:
Now that they are part of the declaration we can remove these 2 lines
```
parsed[:abstract] = false
parsed[:params] = {}
```

Reply:
nice catch, fixed!